tests/test_solver_based_model.py

```diff
@@ -252,6 +252,22 @@ def test_set_bounds_scenario_3(self):
             self.assertEqual(reac.lower_bound, -11)
             self.assertEqual(reac.upper_bound, 2)
 
+        def test_set_bounds_scenario_4(self):
+            reac = self.model.reactions.ACALD
+            reac.knock_out()
+            reac.lower_bound = 2
+            self.assertEqual(reac.lower_bound, 2)
+            self.assertEqual(reac.upper_bound, 2)
```

Comment:
this upper_bound change happens implicitly, which is good and all, could we just have the unit-tests a bit more explicit setting the bounds first, e.g. `reac._lower_bound = 0; reac._upper_bound = 1` first as this would also make us safe to changes in the test model later on